lib/metadata/wrapper.js

```diff
@@ -227,6 +227,18 @@ const metadata = {
         }
         return client.getDiskUsage(cb);
     },
+
+    countItems: (log, cb) => {
+        if (!client.countItems) {
+            log.debug('returning zero item counts as fallback', { implName });
```

Comment:
Yes, since it's management code, I'd rather have the error handling be for actual errors are are retryable, not for unsupported configurations that would never work anyway.